src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAddAudioMetaDataIntTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.darts.audio.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.transaction.Transactional;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import uk.gov.hmcts.darts.audio.model.AddAudioMetaDataRequest;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.net.URI;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@ActiveProfiles({"intTest", "h2db"})
+@AutoConfigureMockMvc
+class AudioControllerAddAudioMetaDataIntTest extends IntegrationBase {
```

Comment:
You've got 2 new integrationTest classes added - is this the one you want to keep - guessing so because of your OAS operation being `addAudioMetaData`?

Think Metadata might be one word - but not essential to change that unless @hemantasharma1129 asks

Reply:
Im happy to change it to be consistent. ACCEPT.